src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.html

```diff
@@ -93,6 +93,12 @@
         </a>
 
         <ul class="dropdown-menu">
+          <li routerLinkActive="active">
+            <a i18n
+               class="dropdown-item"
+               routerLink="/block/rbd">RBDs</a>
```

Comment:
Nit: why not just ```Images``` instead of ```RBDs```? It's already under the ```Block``` drop-down so the "B" becomes redundant.